src/main/java/org/waarp/openr66/protocol/networkhandler/NetworkServerHandler.java

```diff
@@ -274,7 +274,8 @@ public void channelRead0(ChannelHandlerContext ctx, NetworkPacket msg) throws Ex
             packet.clear();
             return;
         }
-        logger.debug("GET MSG: " + packet.getCode());
+        logger.debug("GET MSG: {}" + packet.getCode());
+        logger.trace("MSG: {}" + packet);
```

Comment:
if using `{}`, then you should not use `+` but a comma, right ?
    `logger.trace("MSG: {}", packet);`

Reply:
fixed in be173d7